assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -16,345 +16,40 @@
-								{ __(
-									'Google Analytics 4 Code Snippet',
-									'google-site-kit'
-								) }
-							</h5>
-							<p className="googlesitekit-settings-module__meta-item-data">
-								{ useGA4Snippet && (
-									<span>
-										{ __(
-											'Snippet is inserted',
-											'google-site-kit'
-										) }
-									</span>
-								) }
-								{ ! useGA4Snippet && (
-									<span>
-										{ __(
-											'Snippet is not inserted',
-											'google-site-kit'
-										) }
-									</span>
-								) }
-							</p>
-						</div>
-					</div>
-				</Fragment>
-			) }
+			<GA4SettingsView />
 
-			{ useSnippet && ampMode !== 'primary' && (
-				<div className="googlesitekit-settings-module__meta-items">
-					<div className="googlesitekit-settings-module__meta-item">
-						<h5 className="googlesitekit-settings-module__meta-item-type">
-							{ __(
-								'IP Address Anonymization',
-								'google-site-kit'
-							) }
-						</h5>
-						<p className="googlesitekit-settings-module__meta-item-data">
-							{ anonymizeIP && (
-								<span>
-									{ __(
-										'IP addresses are being anonymized',
-										'google-site-kit'
-									) }
-								</span>
-							) }
-							{ ! anonymizeIP && (
-								<span>
-									{ __(
-										'IP addresses are not being anonymized',
-										'google-site-kit'
-									) }
-								</span>
-							) }
-						</p>
-					</div>
-				</div>
-			) }
+			<IPAnonymizationSettingsView />
 
-			<div className="googlesitekit-settings-module__meta-items">
-				<div className="googlesitekit-settings-module__meta-item">
-					<h5 className="googlesitekit-settings-module__meta-item-type">
-						{ __( 'Excluded from Analytics', 'google-site-kit' ) }
-					</h5>
-					<p className="googlesitekit-settings-module__meta-item-data">
-						{ !! trackingDisabled.length &&
-							trackingDisabled
-								.map(
-									( exclusion ) =>
-										trackingExclusionLabels[ exclusion ]
-								)
-								.join(
-									_x(
-										', ',
-										'list separator',
-										'google-site-kit'
-									)
-								) }
-						{ ! trackingDisabled.length &&
-							__(
-								'Analytics is currently enabled for all visitors',
-								'google-site-kit'
-							) }
-					</p>
-				</div>
-			</div>
+			<ExcludeFromAnalyticsSettingsView />
 
-			{ canUseSnippet && useSnippet && (
-				<div className="googlesitekit-settings-module__meta-items">
-					<div className="googlesitekit-settings-module__meta-item">
-						<h5 className="googlesitekit-settings-module__meta-item-type">
-							{ __( 'Ads Conversion ID', 'google-site-kit' ) }
-						</h5>
-						<p className="googlesitekit-settings-module__meta-item-data">
-							{ adsConversionID ? (
-								<DisplaySetting value={ adsConversionID } />
-							) : (
-								__( 'None', 'google-site-kit' )
-							) }
-						</p>
-					</div>
-				</div>
-			) }
+			<AddConversionIDSettingsView />
```

Comment:
`IPAnonymizationSettingsView`, `ExcludeFromAnalyticsSettingsView`, and `AddConversionIDSettingsView` components can be combined into a single component.